audioplayer.cpp

```diff
@@ -0,0 +1,95 @@
+#include "audioplayer.h"
+
+
+void AudioPlayer::getFileNames(const string &folder, vector<string>) {
```

Comment:
Are you sure that this is compiling? `vector<string>` needs a variable name, and you don't have it right now.

Reply:
> Are you sure that this is compiling? `vector<string>` needs a variable name, and you don't have it right now.

This argument is even does not used in code. That's why it is compiling, my bad. Therefore I'll don't fix issue in this scope right now, I'll refactor all functions anyway.